webapp/src/auth/auth.guard.ts

```diff
@@ -20,10 +19,9 @@ export class AuthGuard implements CanActivate {
     }
     try {
       const payload = await this.jwtService.verifyAsync(token, {
-        secret: jwtConstants.secret,
+        secret: process.env.SECRET,
```

Comment:
`SECRET` is maybe a little to generic, `JWT_SECRET` maybe? Not mandatory to merge